src/export/InstanceExporter.ts

```diff
@@ -314,6 +314,17 @@ export class InstanceExporter implements Fishable {
     // Once all rules are set, we should ensure that we did not add a duplicate profile URL anywhere
     if (instanceDef.meta?.profile) instanceDef.meta.profile = uniq(instanceDef.meta.profile);
 
+    // If we added a versioned profile that's the same as the InstanceOf profile, remove the InstanceOf profile
+    const versionedInstanceOf =
+      instanceDef.meta?.profile?.filter(profile =>
+        profile.startsWith(`${instanceOfStructureDefinition.url}|`)
+      ) ?? [];
```

Comment:
Since this is really just checking to see if at least one item in the array meets the criteria, I think it would be better to use `some`:
```ts
const hasVersionedInstanceOf =
      instanceDef.meta?.profile?.some(profile =>
        profile.startsWith(`${instanceOfStructureDefinition.url}|`)
      );
```